docs/Customization/Incorporating-HLS.rst

```diff
@@ -0,0 +1,69 @@
+.. _incorporating-hls:
+
+Incorporating HLS
+============================
+
+High Level Synthesis (HLS) is a method for iterating quickly on 
+different hardware algorithms that automatically generates an RTL 
+circuit to match a specification in a high level language like C.
+
+Here, we will integrate an HLS-generated accelerator that computes 
+the Great Common Denominator (GCD) of two integers. This tutorial 
+builds on the sections :ref:`mmio-accelerators` and 
+:ref:`incorporating-verilog-blocks`. The code for this example can 
+be found in ``/generators/hls-example``
```

Comment:
I would use `gh-file-ref` to have it auto-generate a link: https://github.com/ucb-bar/chipyard/blob/0ad87ff5cec66396f3b0de4d639de1b2ef7221c5/docs/Simulation/FPGA-Accelerated-Simulation.rst#L26